qiita_db/support_files/patches/24.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE qiita.reference ADD sortmerna_indexed_db_filepath bigint  ;
+
+CREATE INDEX idx_reference_2 ON qiita.reference ( sortmerna_indexed_db_filepath ) ;
```

Comment:
This index was created by DBSchema, when I added the foreign key. If you're sure that is not needed. I can remove it w/o a problem :-)